### DISCUSSION TITLE
GitHub account linked but won’t log in.

### DISCUSSION THREAD

**Discussion by Apogu3** [vercel/community#3797] [Account Issue] [answered]
### Question

I am having the error when trying to login to vercel with GitHub, it directs me to a page that says I already have a linked account and to try to sign in with email. When I then try logging in with email I get an error message saying my email won’t let me log in.

**Comment by amyegan** [comment, marked as answer]
Hey @Apogu3. Please make sure you're going to the [log in](https://vercel.com/login) page and not the sign up page if you do have an existing account.

If you still have trouble logging in, please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy. Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. 🙇
